R/checks.R

```diff
@@ -901,6 +901,15 @@ checkCodingPractice <- function(pkgdir, parsedCode, package_name)
             handleMessage(msg, indent=8)
     }
 
+    # pkg:fun...
+    msg_sc <- checkSingleColon(Rdir)
+    if (length(msg_sc)) {
+        handleError(" Single colon use found in PKG:FUN; use double colon '::'")
```

Comment:
Not sure about this, but is it 'best practice' never to tell the user what they did wrong, but to tell them what they need to do to get it right? Kind of like not planting the seed of bad ideas in their brains...